.github/PULL_REQUEST_TEPLATE.md

```diff
@@ -0,0 +1,55 @@
+<!-- 
+> [!CAUTION]
+> This PR contains migrations! Don't deploy.
+> This PR introduces breaking changes! Don't deploy.
+-->
+
+### What?
+<!-- Describe the changes introduced by this PR on a high level that can be understood by end users. -->
+
+### Why?
+<!-- Describe the motivation behind this PR. -->
+
+### How?
+<!-- Describe the technical implementation of this PR. Be precise for other developers. -->
+
+### Affected Issues & Feature Proposal
+<!-- List the issues that are being resolved by this PR or the feature proposal that is being implemented. -->
+
+### Checklist
+
+<!-- Checklist for all PRs -->
+#### General
+- [ ] I chose a title conforming to the [naming conventions for pull requests](#).
+
+<!-- Checklist for PRs that affect our API-->
+#### AtlasML - API
+- [ ] I updated the API documentation.
+
+#### AtlasML - Machine Learning
+- [ ] I updated the machine learning documentation.
```

Comment:
Do we have a documentation online? If so, we could put the link here for ease of use. Maybe we can note down small items (just one word) for the sections that were affected?

Reply:
We can use Pydoc for documentation. Where we need to write Markdown for documentation. So, in this case we don't need a link or something like that. Everything will be readble in PRs

Reply:
I've already created documentation, see e.g.  #13 but I'm not finished with the deployment yet. Detailed links will follow as soon as a copy is deployed on our own instances.